test/app-tap/module_api.test.lua

```diff
@@ -496,8 +501,82 @@ local function test_box_session_id_matches(test, module)
             'same value')
 end
 
+-- Basic test of `box_iproto_send` correctness: main functional testing is done
```

Comment:
Nit (up to you): we are supposed to do new tests in luatest, no? Why is this tap?

Reply:
From the beginning of this patch series, we decided to add module API related tests to this file, since all module API tests are collected here.